CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+4.0.0 / 2023/11/07
+====================
+- **BREAKING**: This module is now ESM only.
+  - You will need to either migrate your projects to ESM, or import `got-scraping` in an async context via `await import('got-scraping');`
+- **BREAKING**: This module now requires Node.js >= 18
```

Comment:
wait a sec, why is that? our node 16 crawlee tests are passing with this, right?

Reply:
Oh tru, but nodejs 16 is also dead officially... I guess I could drop the requirement back to >= 16 for now